config/application.yml.example

```diff
@@ -3,6 +3,10 @@ default: &default
   SJ_USER_AGENT: Supplejack Harvester v2.0
   FIXTURES_PATH: spec/fixtures
   OTP_ENCRYPTION_KEY: AGoodEncryptionKey
+  SECRET_KEY_BASE: anawesomelysecretsecretkeybase
```

Comment:
Question: Do we require this file?

Reply:
It's just an example for people to use as a template when they set up the application :). A lot of our applications do this.